homeassistant/components/matrix/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow to configure Matrix Bot."""
+import functools
+import logging
+
+from matrix_client.client import MatrixClient, MatrixRequestError
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, CONF_VERIFY_SSL
+
+from .const import CONF_HOMESERVER, DOMAIN
```

Comment:
This is a bug in PyLint. Disable the check:

```suggestion
from .const import CONF_HOMESERVER, DOMAIN  # pylint: disable
```